mlbench/refimpls/pytorch/controlflow/__init__.py

```diff
@@ -0,0 +1,7 @@
+def get_controlflow(options):
```

Comment:
At first, I was thinking about rewriting the controlflow for linear models because some datasets are sparse. But pytorch autogradient is not very friendly to sparse dataset. So I put it aside for the moment. So the current linear model controlflow should be the same as previous one.